.github/workflows/models.yml

```
@@ -0,0 +1,30 @@
+name: Java Models
+
+on: [ workflow_dispatch, pull_request ]
```

This event is only used for testing: 

```suggestion
on: [ workflow_dispatch ]
```